examples/ensembles.ipynb

```diff
@@ -31,7 +31,7 @@
   },
```

Comment:
<p>In addition, you can set <code>weights</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>